src/mWater_collate/clean_mwater_notes.R

```diff
@@ -0,0 +1,59 @@
+#Set up
+library(tidyverse)
+`%nin%` = Negate(`%in%`)
+
+# API Pull of mWater submitted notes
+
+# Grab API url from yml
+# Contact Sam Struthers if you need access
+creds = yaml::read_yaml("src/mWater_collate/mWater_API.yml")
+api_url = as.character(creds["url"])
+
+# Read in from API and tidy for downstream use
+
+# This is basic tidying of data set to:
+# correct datetime from UTC to Denver time
+# correct columns where Other input is allowed (Site, visit type, photos downloaded, sensor malfunction)
+# Add rounded date time
+
+
+all_notes_cleaned <- read_csv(url(api_url),show_col_types = FALSE)%>%
+  mutate(
+    #start and end dt comes in as UTC -> to MST
+    start_DT = with_tz(ymd_hms(start_dt), tz = "MST"),
+    end_dt = with_tz(ymd_hms(end_dt), tz = "MST"),
+    malfunction_end_dt = with_tz(ymd_hms(malfunction_end_dt), tz = "MST"),
```

Comment:
question: where is malfunction_end_dt being created?

Reply:
In Mwater, it is a user input